monitor/pom.xml

```diff
@@ -10,8 +10,8 @@
 	<properties>
 		<org.slf4j-version>1.7.5</org.slf4j-version>
 		<org.springframework-version>5.0.0.RELEASE</org.springframework-version>
-		<spring-security-version>4.2.3.RELEASE</spring-security-version>
-		<admin.password>admin</admin.password>
+		<spring-security-version>5.2.2.RELEASE</spring-security-version>
```

Comment:
what about all the other dependencies? there several in here where there are newer versions available.